models/AddressesModel.php

```diff
@@ -22,21 +20,9 @@ class AddressesModel extends AbstractModel {
         
         /**
          * {@inheritDoc}
-         * 
-         * @return AddressEntity[]
          */
-        public function getList(int $limit, int $offset):array {
-            $connection = $this->connectToDB();
-
-            $query      = (new SelectQueryBuilder($this->getTableName()))
-                            ->whereAnd("`address_is_deleted` = 0")
-                            ->limit($limit, $offset)
-                            ->build();
-
-            $result        = $connection->query($query->getQueryString());
-            $addressesList = $result->fetch_all(MYSQLI_ASSOC);
-
-            return $addressesList;
+        protected function getDomainName():string {
+            return "address";
```

Comment:
Abstract function implementation. It is supposed that when the 'domains' branch will be ready this method will also be transfered to AbstractModel due to the generalisation.